mocha-multi.js

```diff
@@ -49,6 +53,35 @@ function MochaMulti(runner, options) {
   }
 }
 
+util.inherits(MochaMulti, Base);
```

Comment:
Ah, you are right, it isn't necessary at all.  I was looking at other reporters as a point of reference and thought it might be needed at some point, but it isn't needed at all, good catch.